src/endpoint/devices.ts

```diff
@@ -561,8 +561,8 @@ export class DevicesEndpoint extends Endpoint {
 	 * @param id
 	 * @param deviceEvents
 	 */
-	public sendEvents(id: string, deviceEvents: DeviceEventList): void {
-		this.client.post(`${id}/events`, deviceEvents)
+	public async sendEvents(id: string, deviceEvents: DeviceEventList): Promise<void> {
+		await this.client.post(`${id}/events`, deviceEvents)
```

Comment:
Since the return type here is `void`, I think at most, clients might be attempting error handling with `try/catch` which would not be functioning like it should. Making it a Promise shouldn't cause compiler errors, so we would just be opening up the opportunity to fix existing bugs.

For that reason, I think the commit type for this should be `fix` instead of `chore`.

Reply:
Sounds good...updated commit message.